### PR TITLE
Code Review comments implemented

### PR DESCRIPTION
**Pull request by rahulnlb1**
I have implemented the code review comments. Could you please check.

### CODE REVIEW

File: app/modules/util/config/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { env } from 'process';
```
Comment (kashyaprahul94) [inline app/modules/util/config/index.ts]: You tend to love OOP a lot :D 

A simpler way could have been - 

```
const DB_HOST: string = env.DB_HOST || '';
const PORT: string = env.PORT || '3000';

export { DB_HOST, PORT };
```

Any importer, can just select what env variable they need - 

```
import { PORT } from "./config";
```

File: app/modules/project/routes/project-service.ts [hunk 2]
```diff
@@ -21,9 +16,8 @@ export class ProjectService {
         return await projectModel.find({ ID: projectID });
     };
 
-    public update = async (reqBody: Object): Promise<any> => {
-        const data = new projectModel(reqBody);
-        return await data.updateOne(data);
+    public update = async (reqParam: string, reqBody: Object): Promise<any> => {
+        return await projectModel.updateOne({ ID: reqParam }, reqBody);
```
Comment (kashyaprahul94) [inline app/modules/project/routes/project-service.ts]: You do not need to `await` here, you can just return a promise from here, which would be resolved on the caller.

File: app/modules/project/models/index.ts [hunk 3]
```diff
@@ -1 +1,3 @@
-export * from './project-model';
```
Comment (kashyaprahul94) [inline app/modules/project/models/index.ts]: Do you really need this file ?

File: app/app.ts [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
 import * as Express from 'express';
 import * as Mongoose from 'mongoose';
 import * as BodyParser from 'body-parser';
 import * as Morgan from 'morgan';
 
 import { ProjectRoute } from './modules/project';
-
-const port: string = env.PORT || '3000';
-const mongoURL: string = env.DB_HOST || '';
+import { Config } from './modules/util/config';
```
Comment (kashyaprahul94) [inline app/app.ts]: See my comment below for `config.ts`

File: app/modules/project/routes/project-service.ts (hunk 2 above)
Comment (kashyaprahul94) [reply app/modules/project/routes/project-service.ts]: You can remove `await` from everywhere in the file, you can keep `async` though.

File: app/modules/project/routes/project-service.ts [hunk 5]
```diff
@@ -5,22 +5,22 @@ export class ProjectService {
...
     };
 
     public get = async (): Promise<ProjectInteface[]> => {
-        return await projectModel.find({});
+        return projectModel.find({});
     };
 
     public getById = async (projectID: string): Promise<ProjectInteface[]> => {
-        return await projectModel.find({ ID: projectID });
+        return projectModel.find({ ID: projectID });
```
Comment (kashyaprahul94) [inline app/modules/project/routes/project-service.ts]: You could use `findOne` and return type can be `Promise<ProjectInterface>`

Comment (kashyaprahul94) [reply app/modules/project/routes/project-service.ts]: One more thing, conventionally interfaces starts with `I` meaning `ProjectInterface` could become `IProject` which represents the domain, and based on that, each and every other entity would be created, such as Mongoose schema.

**Comment by rahulnlb1** [comment]
Did I make the actual use of ESLint?

**Pull request merged**